fastlane/Fastfile

```diff
@@ -10,13 +10,17 @@ platform :ios do
       build_number: ENV["FASTLANE_BUILD_NUMBER"],
       xcodeproj: "./Samples/iOS-Swift/iOS-Swift.xcodeproj"
     )
+    increment_build_number(
+      build_number: ENV["FASTLANE_BUILD_NUMBER"],
+      xcodeproj: "./Samples/iOS-Swift_xcodegen.xcodeproj"
+    )
   end
 
   # The version for all Info.plist must be a period-separated list of at most three non-negative integers
   # for App Store Connect. If we are on a preview we to remove these suffixes.
   lane :remove_preview_version_suffixes do
     version = get_version_number(
-      xcodeproj: "./Samples/iOS-Swift/iOS-Swift.xcodeproj",
+      xcodeproj: "Samples/iOS-Swift_xcodegen.xcodeproj",
```

Comment:
`m`: At some parts of the PR you are adding `iOS-Swift_xcodegen` in addition for `iOS-Swift`. Here you are replacing it. Please be consistent.

Reply:
See https://github.com/getsentry/sentry-cocoa/pull/5115#discussion_r2074179291. 

If something is broken, please point it out. Otherwise you may think you are looking at inconsistency, but instead it's just a necessary change.